app/views/pages/flash_help.html.erb

```diff
@@ -0,0 +1,65 @@
+<div class="page container flash-help">
```

Comment:
This is missing a closing div? You got 9 opening div tags but only 8 closing div tags

Reply:
Page should also be i18n?

Reply:
Thanks for catching the closing div (would have not been missed if the page had indented all of the divs, which is what it does not).

Regarding i18n, I've moved the partial to be a 'en' partial. At this point, none of the other static content pages use i18n, so I don't think it's worth breaking the page into separate translation strings.